docs/user/05-60-kyma-bindings.md

```diff
@@ -16,32 +31,87 @@ X-Broker-API-Version: 2.14
+|------------------------|---------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **expiration_seconds** | `600`   | Specifies the duration (in seconds) for which the generated kubeconfig is valid. If not provided, the default value of `600` seconds (10 minutes) is used, which is also the minimum value that can be set. The maximum value that can be set is `7200` seconds (2 hours).                                             |
+
+After unmarshaling the data is validated against allowed parameter values, which includes checks of expiration value range, database record existence, parameters mutations verification, expiration of exising bindings and binding number limits. The first of the checks is a verification of the expiration value. Minimum and maximum limits are configurable and, by default, set to 600 and 7200 seconds respectively. After that KEB checks if the binding already exists. The binding in the database is identified by Kyma instance ID and binding ID passed as a path query parameter. If the binding exists, KEB checks mutation of the parameters of the existing binding. The Open Service Broker API requires a create binding request to fail if an object has been already created and the request contains different parameters. Next, if the found binding is not expired, KEB returns it in the response. At this point, the flow gets back to the execution path of the process where no bindigs exist in database at all. Not matter if the binding exists or not the last step in the request validation is the verification of bindings number limit. Every instance is allowed to create a limited number of bindings. The limit is configurable and by default set to 10. If the bindings limit is not exceeded, KEB proceeds to the next phase of the process - binding creation.  
+
+
+<!-- Binding Creation -->
+In the binding creation phase, KEB creates a Service Binding object and generates a kubeconfig file with a JWT token. The kubeconfig file is valid for a specified time period, which is defaulted of set in the request body. The first step in this part is to check again if an expired binding exists in the database. This check is done in an implicit DB insert statement. The query will fail for expired but existing bindings because of primary key being defined on the instance and binding IDs and not expiration date. This will be the case until the expired binding is removed from the database by the cleanup job. 
```

Comment:
DB insert statement is not `implicit` but it implicitly checks if record already exists.